src/gear3/expander.nim

```diff
@@ -402,8 +409,21 @@ proc traverseExpr(e: var EContext; c: var Cursor) =
     case c.kind
     of EofToken: break
     of ParLe:
-      e.dest.add c
-      inc nested
+      if pool.tags[c.tag] == "curlyConstr":
```

Comment:
"curlyConstr" is not in gear3's input language either. (I need to write a spec...)

Reply:
Sure, I will assume the input to be `ranges`